core/trino-main/src/main/java/io/trino/operator/aggregation/MapAggregationFunction.java

```diff
@@ -40,10 +40,11 @@ private MapAggregationFunction() {}
     public static void input(
             @AggregationState({"K", "V"}) MapAggregationState state,
             @BlockPosition @SqlType("K") Block key,
+            @BlockIndex int keyPosition,
             @SqlNullable @BlockPosition @SqlType("V") Block value,
```

Comment:
We could make that change everywhere.  I think we put `@SqlNullable` first because rare, but we could make this change (looks like there are only 20 uses)